include/abb_librws/v1_0/rw/retcode.h

```diff
@@ -0,0 +1,44 @@
+#include <abb_librws/rws.h>
+#include <abb_librws/v1_0/rws_client.h>
+
+#include <string>
+
+
+namespace abb :: rws :: v1_0 :: rw
+{
+    using namespace rws::rw;
+}
+
+
+namespace abb :: rws :: v1_0 :: rw :: retcode
+{
+    /**
+     * \brief A structure for containing response of recode resource.
+     */
+    struct RetcodeInfo
+    {
+        int code;                   /** \brief The error code or success code as number */
+        std::string name;           /** \brief The error code or success code as string */
+        std::string severity;       /** \brief Success, Warning or Error */
+        std::string description;    /** \brief Short description of the error code */
+    };
+
+    /**
+     * \brief Error info containing RetcodeInfo struct.
+     */
+    using ErrorCodeInfo = boost::error_info<struct RetcodeInfoInfo, RetcodeInfo>;
```

Comment:
We don't attach `ErrorCodeInfo` to exceptions that we throw from `abb_librws`, right? Then I don't think `ErrorCodeInfo` should be a part of `abb_librws`.

Reply:
```suggestion
```